lang/LangSource/SC_TerminalClient.cpp

```diff
@@ -527,12 +528,13 @@ void SC_TerminalClient::startInputRead()
 		else {
 			DWORD bytes_transferred;
 
+			postfl("Going to ReadFile()\n");
```

Comment:
Yes, I left some debugging info, while @bagong was testing in case there were problems. I will remove and commit now.